Update installation.md

tcskiran opened the pull request:
I have read these commands
```bash
sudo apt-get update -qq
sudo apt-get -qq -y install podman
```
and have tried using
```bash
man apt-get
```
to understand -qq command but couldn't find anything.

Thinking that maybe you are trying to use quiet I have changed -qq to -q

lsm5 commented:
skipped raspbian until https://github.com/containers/libpod/issues/4903 is fixed.

lsm5 commented:
btw, I notice, buildah.io's install link points to the buildah github repo itself, while podman.io has this separate page. Maybe just me, but it'd be great to have a consistent look-and-feel.

lsm5 commented:
raspbian build succeeded just now, I'll update this PR..

TomSweeneyRedHat commented:
@lsm5 it's not just you.  @jberkus was able to spend some time making Podman.io more user-friendly as it generally has more traffic than Buildah.io.  I've not done the same in Buildah.io partially because I like having the install instructions in one place.  But perhaps the longer term answer is to just have the install instructions only on the *.io site and add links in the respective GitHub repo's for both.

TomSweeneyRedHat commented:
The current verbiage LGTM, will merge in after you update the raspbian stuff too.  Thanks!

lsm5 commented:
@TomSweeneyRedHat ready to go..

TomSweeneyRedHat commented:
and forgot, thanks @lsm5 !

tcskiran committed:
Update installation.md

I have tried using
```bash
man apt-get
```
to understand -qq command but couldn't find anything.

Thinking that maybe you are trying to use quiet I have changed -qq to -q

rhatdan commented:
I am no apt expert, but I did find this comment.

https://serverfault.com/questions/644180/what-does-qq-argument-for-apt-get-mean

rhatdan commented:
Thanks @tcskiran but it looks from the man page the -qq is valid.
